velox/core/PlanNode.h

```diff
@@ -532,6 +532,27 @@ class AggregationNode : public PlanNode {
       bool ignoreNullKeys,
       PlanNodePtr source);
 
+  /// @param globalGroupingSets If the AggregationNode is over a GroupId node
```

Comment:
nit: Group IDs of the global grouping sets produced by the preceding GroupId node.